### PR TITLE
Page/appliances/fix

### PR DESCRIPTION
**Pull request by IanCStewart**

### CODE REVIEW

File: src/css/pages/appliances.css [hunk 1]
```diff
@@ -21,13 +21,23 @@ fieldset {
...
 }
 
 main {
   margin: 1em;
 }
 
+main a {
+  color: #5C637A;
+  opacity: .65;
+  transition: .3s ease-in-out;
```
Comment (dandevri) [inline src/css/pages/appliances.css]: This transitions all properties of the anchor. Could cause problems when other properties get modified in states

File: src/js/appliances.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
-var getFieldset = document.getElementById('add').addEventListener('click', addFieldset);
-var getButton = document.getElementById('remove').addEventListener('click', removeFieldset);
+document.getElementById('add').addEventListener('click', addFieldset);
+document.getElementById('remove').addEventListener('click', removeFieldset);
 
 function addFieldset() {
-  document.getElementsByTagName('main')[0].innerHTML +=
-  `<fieldset class="aplliances">
+  document.getElementsByClassName('appliances')[document.getElementsByClassName('appliances').length - 1].insertAdjacentHTML('afterend',
```
Comment (dandevri) [inline src/js/appliances.js]: dry

File: views/pages/appliances.ejs [hunk 3]
```diff
@@ -14,26 +14,29 @@
...
+    <a href="/">↤ Go Home</a>
+    <form method="post">
+      <fieldset class="appliances">
+        <div class="field">
+          <label for="appliance">Appliance name</label>
+          <input type="text">
+          <label for="voltage">Voltage <strong>(v)</strong></label>
+          <input type="number" min="0">
+          <label for="energy">Ampere <strong>(a)</strong></label>
+          <input type="number" min="0">
```
Comment (dandevri) [inline views/pages/appliances.ejs]: no ID set, so label doesn't connect to input

File: views/pages/appliances.ejs [hunk 4]
```diff
@@ -14,26 +14,29 @@
...
+          <label for="quantity">Quantity</label>
+          <input type="number" min="0">
+        </div>
+      </fieldset>
+      <div class="buttons">
+        <button type="button" id='add'>✚ Add</button>
+        <button type="button" id='remove'>✖ Remove</button>
       </div>
-    </fieldset>
+      <button type="submit" name="appliance">Submit appliances</button>
```
Comment (dandevri) [inline views/pages/appliances.ejs]: Button doesn't need a name attribute

**Review by dandevri — DISMISSED** [review]

**Review by Mimaaa — APPROVED** [review]
🌮

**Pull request merged**